src/java/org/apache/cassandra/config/DurationSpec.java

```diff
@@ -135,6 +135,25 @@ public TimeUnit unit()
         return unit;
     }
 
+    public static <T extends DurationSpec> T from(String value, Class<T> clazz)
```

Comment:
can we remove?  The only usage can be replaced with `DurationSpec.IntMillisecondsBound::new`